src/tls/main.c

```diff
@@ -121,7 +121,7 @@ main (int argc, char **argv)
       if (cockpit_conf_bool ("WebService", "ClientCertAuthentication", false))
         client_cert_mode = GNUTLS_CERT_REQUEST;
 
-      connection_crypto_init (certfile, client_cert_mode);
+      connection_crypto_init (certfile, client_cert_mode, cockpit_conf_string ("WebService", "ClientCertAuthenticationCA"));
```

Comment:
a separate `const char *` variable would make this look nicer and also give me a hint that you're not accidentally leaking the return value. :)